packages/redux-subspace/src/actions/processAction.js

```diff
@@ -8,9 +8,10 @@
 
 import hasNamespace from '../actions/hasNamespace'
 import isGlobal from '../actions/isGlobal'
+import isReduxSpecific from '../actions/reduxSpecific'
 
 const processAction = (namespace) => (action, callback, defaultValue) => {
-    if (!namespace || isGlobal(action)) {
+    if (!namespace || isGlobal(action) || isReduxSpecific(action)) {
```

Comment:
This case should get a specific unit test.

Reply:
I'll add tests a bit later